src/main/java/frc/robot/constants/StorageConstants.java

```diff
@@ -0,0 +1,8 @@
+package frc.robot.constants;
+
+public class StorageConstants {
+
+    public static final int kStorageMotorPort = 0;
```

Comment:
```suggestion
    public static final int motorPort = 0;
```
don't prefix with k
Also, since we will call is `StorageConstants.motorPort` we don't need the word storage here